Source/QED/qed_wrapper_commons.h

```diff
@@ -0,0 +1,13 @@
+#ifndef WARPX_amrex_qed_wrapper_commons_h_
+#define WARPX_amrex_qed_wrapper_commons_h_
+
+//Common definitions for the QED library wrappers
+
+#include <AMReX_AmrCore.H>
+
+//Sets the decorator for GPU
```

Comment:
In the QED library, all the methods which could conceivably be run on a GPU have a PXRMP_GPU decorator. The user is responsible for setting this parameter to the right value.

Now the QED library can be used either with SI units or with normalized units (c=1, m_e=1, omega=1...).
Units are chosen with a define (PXRMP_WITH_SI_UNITS in this case).
Actually, I am not completely satisfied with this solution. In the future I would like to propose a template-based alternatives, like breit_wheeler_engine<double,units_si> and breit_wheeler_engine<double,units_norm>.

Reply:
>[...] do we really need all of them to be __host__ __device__, i.e., compiled to run on both the CPU and GPU?
Actually no.  The functions decorated  with PXRMP_GPU will only be called in an `amrex::ParallelFor` cycle . I've just corrected this. 

> Ok, I am find with both solutions, i.e., template or #define (I agree the template one is nicer, but I don't think there is a rush here).

I agree. We could discuss that after the coupling is finished.